### PR TITLE
add workaround for charset error when migrate data

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/11718
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/dm-faq.md [hunk 1]
```diff
@@ -392,3 +392,7 @@ You can avoid this error by the following options:
     ```
 
 - Upgrade DM to v2.0.7 or later versions.
+
+## The load unit reports the `Unknown character set` error
+
+TiDB does not support all MySQL characters sets. Therefore, DM reports this error if an unsupported character set is used when creating the table structure during a full import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance. For details about the supported character sets, see .
```
Comment (lance6716) [inline dm/dm-faq.md:398]:
```suggestion
TiDB does not support all MySQL character sets. Therefore, DM reports this error if an unsupported character set is used when creating the table structure during a full import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance. For details about the supported character sets, see [character sets supported by TiDB](/character-set-and-collation.md).
```

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 2]
```diff
@@ -201,3 +201,7 @@ This error occurs usually because the CSV data file does not contain a header (t
 [mydumper.csv]
 header = false
 ```
+
+### `Unknown character set`
+
+TiDB does not support all MySQL characters sets. Therefore, TiDB Lightning reports this error if an unsupported character set is used when creating the table structure during a import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance. For details about the supported character sets, see .
```
Comment (lance6716) [inline tidb-lightning/troubleshoot-tidb-lightning.md:207]:
```suggestion
TiDB does not support all MySQL character sets. Therefore, TiDB Lightning reports this error if an unsupported character set is used when creating the table structure during a import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance. For details about the supported character sets, see [character sets supported by TiDB](/character-set-and-collation.md).
```

**Review by lance6716 — APPROVED** [review]
rest lgtm

File: dm/dm-faq.md [hunk 3]
```diff
@@ -392,3 +392,7 @@ You can avoid this error by the following options:
     ```
 
 - Upgrade DM to v2.0.7 or later versions.
+
+## The load unit reports the `Unknown character set` error
+
+TiDB does not support all MySQL character sets. Therefore, DM reports this error if an unsupported character set is used when creating the table structure during a full import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance.
```
Comment (qiancai) [inline dm/dm-faq.md:398]:
```suggestion
TiDB does not support all MySQL character sets. Therefore, DM reports this error if an unsupported character set is used when creating the table schema during a full import. To bypass this error, you can create the table schema in the downstream in advance using the [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data.
```

File: tidb-lightning/troubleshoot-tidb-lightning.md [hunk 4]
```diff
@@ -201,3 +201,7 @@ This error occurs usually because the CSV data file does not contain a header (t
 [mydumper.csv]
 header = false
 ```
+
+### `Unknown character set`
+
+TiDB does not support all MySQL character sets. Therefore, TiDB Lightning reports this error if an unsupported character set is used when creating the table structure during an import. You can bypass this error by using [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data and creating the table structure in the downstream in advance.
```
Comment (qiancai) [inline tidb-lightning/troubleshoot-tidb-lightning.md:207]: Please check if similar changes need to be made.

File: dm/dm-faq.md [hunk 5]
```diff
@@ -392,3 +392,7 @@ You can avoid this error by the following options:
     ```
 
 - Upgrade DM to v2.0.7 or later versions.
+
+## The load unit reports the `Unknown character set` error
```
Comment (shichun-0415) [inline dm/dm-faq.md:396]:
```suggestion
## Why does the load unit report the `Unknown character set` error?
```

File: tidb-lightning/troubleshoot-tidb-lightning.md (hunk 4 above)
Comment (shichun-0415) [inline tidb-lightning/troubleshoot-tidb-lightning.md:207]:
```suggestion
TiDB does not support all MySQL character sets. Therefore, TiDB Lightning reports this error if an unsupported character set is used when creating the table schema during an import. To bypass this error, you can create the table schema in the downstream in advance using the [character sets supported by TiDB](/character-set-and-collation.md) according to the specific data.
```

File: dm/dm-faq.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (shichun-0415) [reply dm/dm-faq.md]: Fixed in [7a3ee27](https://github.com/pingcap/docs/pull/10974/commits/7a3ee271b216780b8e016083758bc7688c4118bb)

File: tidb-lightning/troubleshoot-tidb-lightning.md (hunk 4 above)
Comment (shichun-0415) [reply tidb-lightning/troubleshoot-tidb-lightning.md]: Fixed in [7a3ee27](https://github.com/pingcap/docs/pull/10974/commits/7a3ee271b216780b8e016083758bc7688c4118bb)

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**